seatunnel-connectors-v2/connector-http/connector-http-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/http/config/HttpConfig.java

```diff
@@ -67,7 +67,7 @@ public class HttpConfig {
     public static final Option<ResponseFormat> FORMAT =
             Options.key("format")
                     .enumType(ResponseFormat.class)
-                    .defaultValue(ResponseFormat.JSON)
+                    .noDefaultValue()
```

Comment:
I think you should not change the default value of `format`. Because it only worked when user configure Http source with `schema` config. Please refer https://github.com/apache/seatunnel/pull/5939/files#diff-7c704c6a3927123b31b41483102f286339da3832309810496f19410c2548f06fR118-R150

Reply:
oh I see. How about add new ResponseFormat `TEXT`? Then set it as default value. It used when no schema config. Also please update the doc. Thanks